nabla/test/swapPool.ts

```diff
@@ -63,6 +63,44 @@ export default async function (environment: TestSuiteEnvironment) {
       await genericTestOnlyOwnerCanSetPoolCap(pool, environment);
     },
 
+    async test_maxCoverageRatioForSwapIn_CheckDefaultValue() {
```

Comment:
These are additional tests from this PR in the Nabla repo: https://github.com/0xamberhq/contracts/pull/76/files#diff-ef61283d9b153c522945399fe4fa316c48029036ca90a2648efa5d5794d5c5ee